src/js/components/helptips/onboardingcompletetip.js

```diff
@@ -73,7 +73,10 @@ export default class OnboardingCompleteTip extends React.Component {
           <p>You&apos;ve now got a good foundation in how to use Mender. Look for more help hints in the UI as you go along.</p>
           What next?
           <div>
-            <a href={`https://docs.mender.io/${AppStore.getDocsVersion()}/getting-started/deploy-to-physical-devices#prepare-the-disk-image`} target="_blank">
+            <a
+              href={`https://docs.mender.io/${AppStore.getDocsVersion() ? `${AppStore.getDocsVersion()}/` : ''}getting-started/deploy-a-system-update-demo`}
```

Comment:
same with this link
https://docs.mender.io/getting-started/on-premise-installation/deploy-a-system-update-demo

Reply:
Thanks for reviewing this - I fixed the link and it should now be good to go! 👌